src/backend/db/models.py

```diff
@@ -19,11 +19,13 @@ class User(Base):
     email = Column(String(50), nullable=False, index=True)
     hashed_password = Column(String(100), nullable=False)
     disabled = Column(Boolean, nullable=False)
+    signup_date = Column(DateTime, nullable=True)
```

Comment:
Можно запилить значение по дефолту, чтобы есть вдруг где будет создаваться клиент (хоть на какой-то странице лендинга) не забыть прописать ему signup_date

```suggestion
    signup_date = Column(DateTime, default=datetime.utcnow, nullable=True)
```

Я ещё при создании Invoce использовал время utc, поэтому похоже нужно прийти к едуному соглашению о том в какой зоне пишем время.